yarn-project/circuits.js/src/types/public_keys.ts

```diff
@@ -41,8 +42,37 @@ export class PublicKeys {
     );
   }
 
-  static empty(): PublicKeys {
-    return new PublicKeys(Point.ZERO, Point.ZERO, Point.ZERO, Point.ZERO);
+  static default(): PublicKeys {
+    // This information is duplicated in noir-protocol-circuits/crates/types/src/public_keys.nr
+    // We use this because empty will produce a point not on the curve.
+    // This is:
+    // "az_null_npk"
```

Comment:
Same as above, we should preserve this as `hash_to_curve("az_null_npk")` as well in the constants file